src/schema/me/create_credit_card_mutation.js

```diff
@@ -34,6 +39,33 @@ export const CreditCardType = new GraphQLObjectType({
   }),
 })
 
+export const CreditCardMutationSuccessType = new GraphQLObjectType({
+  name: "CreditCardMutationSuccess",
+  isTypeOf: data => data.id,
```

Comment:
I'm not a fan of the introspection for `isTypeOf`. Is it possible to inject a `_type: "Error"` or something in the response, and then `isTypeOf: data => data._type === "Error"` kinda thing?

Basically, at the time you are resolving either a card or an error (below), you _know_ what type it is, so why rely on this indirection to determine what type it is, when it can be passed thru directly.

Reply:
Updated the error case below to return a specific type.